elastic/CHANGELOG.md

```diff
@@ -2,250 +2,457 @@
+
 
 ## 1.7.0 / 2018-06-07
 
-* [Added] Package `auto_conf.yaml` for appropriate integrations. See [#1664](https://github.com/DataDog/integrations-core/pull/1664).
-* [Fixed] [FIXED] Ensure base url path isn't removed when admin_forwarder is used. See [#1202](https://github.com/DataDog/integrations-core/pull/1202).
+***Added***: 
+
+* Package `auto_conf.yaml` for appropriate integrations. See [#1664](https://github.com/DataDog/integrations-core/pull/1664).
+
+***Fixed***: 
+
+* [FIXED] Ensure base url path isn't removed when admin_forwarder is used. See [#1202](https://github.com/DataDog/integrations-core/pull/1202).
+
 
 ## 1.6.0 / 2018-05-11
 
-* [FEATURE] Hardcode the 9200 port in the Autodiscovery template. See [#1444](https://github.com/DataDog/integrations-core/issues/1444).
-* [FEATURE] adds `index_stats` to collect index level metrics. See [#1312](https://github.com/DataDog/integrations-core/issues/1312).
+***Added***: 
+
+* Hardcode the 9200 port in the Autodiscovery template. See [#1444](https://github.com/DataDog/integrations-core/issues/1444).
+* adds `index_stats` to collect index level metrics. See [#1312](https://github.com/DataDog/integrations-core/issues/1312).
+
 
 ## 1.5.0 / 2018-02-13
 
-* [IMPROVEMENT] Adds `admin_forwarder` option to keep URL intact when using forwarder. See [#1050](https://github.com/DataDog/integrations-core/issues/1050).
-* [BUG] Fixes bug that causes poor failovers when authentication fails. See [#1026](https://github.com/DataDog/integrations-core/issues/1026).
-* [IMPROVEMENT] Adds `cluster_name` tag to the `elasticsearch.cluster_health` service check. See [#1038](https://github.com/DataDog/integrations-core/pull/1038).
+***Added***: 
+
+* Adds `admin_forwarder` option to keep URL intact when using forwarder. See [#1050](https://github.com/DataDog/integrations-core/issues/1050).
+* Adds `cluster_name` tag to the `elasticsearch.cluster_health` service check. See [#1038](https://github.com/DataDog/integrations-core/pull/1038).
+
+***Fixed***: 
+
+* Fixes bug that causes poor failovers when authentication fails. See [#1026](https://github.com/DataDog/integrations-core/issues/1026).
+
 
 ## 1.4.0 / 2018-01-10
 
-* [BUG] Fix missing fs metrics for elastic >= 5. See [#997](https://github.com/DataDog/integrations-core/pull/997).
+***Fixed***: 
+
+* Fix missing fs metrics for elastic >= 5. See [#997](https://github.com/DataDog/integrations-core/pull/997).
+
 
 ## 1.3.0 / 2018-01-10
 
-* [FEATURE] adds `pshard_graceful_timeout` that will skip pshard_stats if TO. See [#463](https://github.com/DataDog/integrations-core/issues/463)
-* [IMPROVEMENT] get rid of pretty json. See [#893](https://github.com/DataDog/integrations-core/issues/893).
+***Added***: 
+
+* adds `pshard_graceful_timeout` that will skip pshard_stats if TO. See [#463](https://github.com/DataDog/integrations-core/issues/463)
+* get rid of pretty json. See [#893](https://github.com/DataDog/integrations-core/issues/893).
+
```

Comment:
```suggestion

***Fixed***:

* get rid of pretty json. See [#893](https://github.com/DataDog/integrations-core/issues/893).

```

Reply:
Looks like this one got miscategorized, but feel free to ignore this suggestion if not